data/campaigns/fri04.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,280 @@
+}
+reebaud_6 = {
+   title =_("An Old Friend"),
+   body=claus(_("Met up at last"),
+      -- TRANSLATORS: Claus Lembeck – Reebaud 6
+      _([[Only you have a reputation for the most wonderous feats of arms. You are probably the only human alive who can vanquish the evil Dane. I beg you to come South with me and liberate Föhr, or at least to send with me a troop of your far-famed soldiers.]]))
+}
+reebaud_7 = {
+   title =_("An Old Friend"),
+   body=reebaud(_("Met up at last"),
+      -- TRANSLATORS: Reebaud – Reebaud 7
+      _([[Had you arrived a week earlier, I would have agreed to your request instantly. Now, however, a spot of bother has appeared on the horizon. I mean this quite literally; foreigners have come from far away and seek to drive us out and take over our lands as we ourselves did, driven by need, seize them from the hostile people who lived here. Warlike Barbarians, and an imperial scallywag named Murilius who made the most outrageous demands. We defeated them all, and believed to have found peace at last. And for four long years, life was peaceful indeed.]]))
+}
+reebaud_8a = {
+   title =_("An Old Friend"),
+   body=reebaud(_("Met up at last"),
+      -- TRANSLATORS: Reebaud – Reebaud 8a
+      _([[But now, as I was saying, in the arrival of the short period of summer, during which the snow melts in some places and travel to distant places is possible, the strangers appeared and engaged us in battle. They took us by surprise, the life without fighting having made us careless. Many a good soldier fell to their spears, including Hauke, my best friend and most venerable advisor, and Fiete, my only son and heir. Their sneaky battle moves were a novelty to us, though now that we know what we are up against there can be no doubt that we will beat them in the end. But I fear it will still take some time until they are defeated and it is safe for me to travel South with you. For this period, you shall of course receive all the hospitality and courtesy we can offer.]]))
+}
+reebaud_8b = {
+   title =_("An Old Friend"),
+   body=reebaud(_("Met up at last"),
+      -- TRANSLATORS: Reebaud – Reebaud 8b
+      _([[But now, as I was saying, in the arrival of the short period of summer, during which the snow melts in some places and travel to distant places is possible, the strangers appeared and engaged us in battle. They took us by surprise, the life without fighting having made us careless. Many a good soldier fell to their spears, including Hauke, my best friend and most venerable advisor, and Fiete, my only son and heir. Their sneaky battle moves were a novelty to us, though now that we knew what we were up against we managed to beat them in the end. However, all our soldiers are still sore and wounded from the heavy fighting, and they will need some time to recuperate before we can send them into battle again. For this period, you shall of course receive all the hospitality and courtesy we can offer.]]))
+}
+reebaud_9a = {
```

Comment:
Wouldn't it make sense to at least add a short sentence of Claus expressing his condolescence about Reebauds losses, at the beginning of each answer 9